packages/core/src/lib/banner/__tests__/banner.spec.svelte

```diff
@@ -17,6 +17,6 @@ export let extraClasses: cx.Argument = '';
 >
   <svelte:fragment slot="title">This is the title.</svelte:fragment>
   <svelte:fragment slot="subtitle">This is the subtitle.</svelte:fragment>
-  <svelte:fragment slot="message">This is the message.</svelte:fragment>
   <svelte:fragment slot="action">This is the action.</svelte:fragment>
+  <svelte:fragment slot="actionEmphasize">This is the button.</svelte:fragment>
```

Comment:
are we doing `actionEmphasize` case or `action-emphasize` case with other slots? just curious if this one is different